### PR TITLE
fix: apply computed at statusList for translation

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA
https://pyengine.atlassian.net/browse/QA-718

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/collector-history/CollectorHistoryPage.vue [hunk 1]
```diff
@@ -54,13 +54,13 @@ const fields: DataTableField[] = [
...
 ];
-const statusList = [
+const statusList = computed(() => [
     { name: JOB_SELECTED_STATUS.ALL, label: i18n.t('INVENTORY.COLLECTOR.HISTORY.ALL') },
     { name: JOB_SELECTED_STATUS.PROGRESS, label: i18n.t('INVENTORY.COLLECTOR.HISTORY.IN_PROGRESS') },
     { name: JOB_SELECTED_STATUS.SUCCESS, label: i18n.t('INVENTORY.COLLECTOR.HISTORY.SUCCESS') },
     { name: JOB_SELECTED_STATUS.FAILURE, label: i18n.t('INVENTORY.COLLECTOR.HISTORY.FAILURE') },
     { name: JOB_SELECTED_STATUS.CANCELED, label: i18n.t('INVENTORY.COLLECTOR.HISTORY.CANCELED') },
-];
+]);
```
Comment (sulmoJ) [inline apps/web/src/services/asset-inventory/collector/collector-history/CollectorHistoryPage.vue:63]: Suggestion: I think you don't need this code!

Comment (sulmoJ) [reply apps/web/src/services/asset-inventory/collector/collector-history/CollectorHistoryPage.vue:63]: How about applying computed to state.statusList below code?

Comment (skdud4659) [reply apps/web/src/services/asset-inventory/collector/collector-history/CollectorHistoryPage.vue:63]: i removed it!

File: apps/web/src/services/asset-inventory/collector/collector-history/CollectorHistoryPage.vue [hunk 2]
```diff
@@ -94,6 +94,13 @@ const state = reactive({
     isDomainOwner: computed(() => store.state.user.userType === 'DOMAIN_OWNER'),
     selectedStatus: 'ALL',
     items: [] as any[],
+    statusList: [
+        { name: JOB_SELECTED_STATUS.ALL, label: i18n.t('INVENTORY.COLLECTOR.HISTORY.ALL') },
+        { name: JOB_SELECTED_STATUS.PROGRESS, label: i18n.t('INVENTORY.COLLECTOR.HISTORY.IN_PROGRESS') },
+        { name: JOB_SELECTED_STATUS.SUCCESS, label: i18n.t('INVENTORY.COLLECTOR.HISTORY.SUCCESS') },
+        { name: JOB_SELECTED_STATUS.FAILURE, label: i18n.t('INVENTORY.COLLECTOR.HISTORY.FAILURE') },
+        { name: JOB_SELECTED_STATUS.CANCELED, label: i18n.t('INVENTORY.COLLECTOR.HISTORY.CANCELED') },
+    ],
```
Comment (sulmoJ) [inline apps/web/src/services/asset-inventory/collector/collector-history/CollectorHistoryPage.vue]: How about applying computed to state.statusList here?

**Review by sulmoJ — APPROVED** [review]
LGTM!

**Pull request merged**